### PR TITLE
my first baseline model comparison

### PR DESCRIPTION
**Pull request by JaydeepBhat**
I made a simple baseline model with VGG16, DenseNet121 and EfficientNetB7 comparison

### CODE REVIEW

**Review by marlomm — APPROVED** [review]
Nice work, well structured notebook!
Not really sure about the distribution of classes, I think it should count 100 images per class not 1. And I'm surprised that the VGG16 achieves such a low validation accuracy in your version and still searching for the reason. 
But overall, thank you, approved! :)

**Comment by wilsoncooked** [comment]
Really nice graphs to display each models performance and clean functions!! 🐶

**Review by wilsoncooked — APPROVED** [review]

**Comment by marlomm** [comment]
I think you forgot to preprocess the test images and it would be slightly better to compare the models performances on the test instead of the validation images but should not make a large difference when you preprocess the test images. 
What is still puzzling me is that the output of the last epoch of your training is loss: 4.7860 - accuracy: 0.0100 - val_loss: 4.7916 - val_accuracy: 0.0046 but then you have a val_accuracy of 0.2912 when you run evaluate manually on the same images.

**Comment by JaydeepBhat** [comment]
Thank you for your positive feedback. I have made changes in the code and request to review/approve these changes.

**Review by marlomm — APPROVED** [review]
Well done, looks good! 🐶
